scripts/west_commands/runners/nrfjprog.py

```diff
@@ -106,6 +106,9 @@ def do_exec_op(self, op, force=False):
                 cmd.append('--reset')
             if _op['kind'] == 'RESET_PIN':
                 cmd.append('--pinreset')
+        elif op_type == 'erase':
+                kind = _op['kind']
```

Comment:
The linter error about this being unused looks legit

Reply:
Indeed! Fixed.